client/lib/signup/step-actions/index.js

```diff
@@ -55,6 +55,7 @@ import flows from 'calypso/signup/config/flows';
 import steps, { isDomainStepSkippable } from 'calypso/signup/config/steps';
 import { isEligibleForPageBuilder, shouldEnterPageBuilder } from 'calypso/lib/signup/page-builder';
 import { fetchSitesAndUser } from 'calypso/lib/signup/step-actions/fetch-sites-and-user';
+import isBlankCanvasDesign from '@automattic/design-picker';
```

Comment:
This breaks signing up because it imports the DesignPicker component.

```suggestion
import { isBlankCanvasDesign } from '@automattic/design-picker';
```

Reply:
Good catch! Yeah, our world would be a really scary place without TypeScript 😬